core/src/apps/cardano/sign_tx.py

```diff
@@ -120,20 +121,17 @@ async def _sign_standard_tx(
         await _show_standard_tx(ctx, keychain, msg)
 
         # sign the transaction bundle and prepare the result
```

Comment:
This comment here and the one in `_sign_stake_pool_registration_tx` aren't accurate anymore.